config.ru

```diff
@@ -8,21 +14,70 @@ conn = PG::Connection.open(ENV["DATABASE_URL"])
 table = convert_keys(conn.exec('SELECT * from accounts'))
 
 app = -> (env) do
-   accounts = table.map do |line|
-     tds = line.map { |key, val| "<td>#{val}</td>" }.join
-     "<tr>#{tds}</tr>"
-   end
-   body = "<h1> Status account</h1>
-     <table>
-         <tr>
-           <th>Type</th>
-           <th>Name</th>
-           <th>Currency</th>
-           <th>Amount</th>
-         </tr>
-         #{accounts.join}
-   </table>"
-   [ 200, { "Content-Type" => "text/html" }, [body] ]
+  if env["REQUEST_METHOD"] == "GET"
+    case env["REQUEST_PATH"]
+    when "/"
+      accounts = table.map do |line|
+        tds = line.map { |key, val| "<td>#{val}</td>" }.join
+        "<tr>#{tds}</tr>"
```

Comment:
this code is also not safe as can have [xss](https://www.netsparker.com/blog/web-security/preventing-xss-ruby-on-rails-web-applications/) attack